### DISCUSSION TITLE
一些繁体字识别不出来

### DISCUSSION THREAD

**Discussion by Yxiaofa** [nju-lug/NJUThesis#235] [Q&A] [answered]
![微信截图_20240226183315](https://github.com/nju-lug/NJUThesis/assets/98028423/2bba4949-3271-47b8-9974-4a1ef995b1cf)

**Comment by Yxiaofa** [comment]
僆-这个字

**Comment by atxy-blip** [comment, marked as answer]
见 <https://github.com/nju-lug/NJUThesis/issues/93>。
